dark-mode.css

```diff
@@ -336,6 +342,22 @@ body.dark-mode .permissions-manager .permission-grid .id-styler {
 	color: var(--info-font-color);
 }
 
+body.dark-mode .rc-header--room {
+	box-shadow: 0px 0px 3px var(--color-gray-medium);
```

Comment:
In light mode, the box shadow on this div is pretty subtle, but this setting is pretty bold. What would you think of either thinning the shadow or darkening the color?

Here's the current look:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/39106297/73668927-a1079200-4674-11ea-806e-b9e1df6f70ef.png">

Here's with `box-shadow: 0px 0px 1px var(--color-gray-medium);`:
<img width="208" alt="image" src="https://user-images.githubusercontent.com/39106297/73668994-c0062400-4674-11ea-80fa-33288cf1a6b1.png">

And here's with `box-shadow: 0px 0px 3px var(--color-dark-medium);`:
<img width="206" alt="image" src="https://user-images.githubusercontent.com/39106297/73669063-d9a76b80-4674-11ea-8b06-9fdeeb620ec7.png">

Let me know what you and your users think.

Reply:
Hrm, I had tried the 2nd look first off, but I felt it looked too sharp and needed a bit more feathering, which is why I opted for the 3px blur on the current one. The last one is quite nice and I like the subtleness. I hadn't thought to just darken the color, which is brilliant

Reply:
Yeah, after testing I wholeheartedly love option 3

Reply:
Great! Unless you want to try any different shades, you can accept this suggestion:

```suggestion
	box-shadow: 0px 0px 3px var(--color-dark-medium);
```